flink-runtime-web/pom.xml

```diff
@@ -91,6 +91,14 @@ under the License.
 			<scope>test</scope>
 		</dependency>
 
+		<dependency>
```

Comment:
This is required as `PackagedProgram` can only load a valid JAR with a properly set main-class entry and reusing this seemed the easiest way.